db/crud.go

```diff
@@ -421,7 +421,7 @@ func (db *Database) Get1xRevAndChannels(docid string, revid string, listRevision
 	roleAccess = doc.RoleAccess
 	sequence = doc.Sequence
 	flags = doc.Flags
-	gotRevID = doc.RevID
+	//gotRevID = doc.RevID
```

Comment:
Why is this being changed?

Reply:
As discussed - return `doc.CurrentRev` when revid is empty for the `explicitDocIDs` handling of `_all_docs`